crates/factor-variables/src/provider.rs

```diff
@@ -1,7 +1,11 @@
-use std::{collections::HashMap, sync::Arc};
+mod env;
+mod statik;
 
-use serde::{de::DeserializeOwned, Deserialize};
-use spin_expressions::{async_trait::async_trait, Key, Provider};
+pub use env::EnvVariables;
+pub use statik::StaticVariables;
+
+use serde::de::DeserializeOwned;
+use spin_expressions::Provider;
 use spin_factors::anyhow;
 
 pub trait MakeVariablesProvider: 'static {
```

Comment:
Do you think we should switch to the runtime resolver pattern here?

Reply:
That's the next PR 😄

Reply:
https://github.com/fermyon/spin/compare/factors...untoml-variables